### PR TITLE
Kbin sync 8: Transfer request for ownership of abandoned magazines 

### PR DESCRIPTION
**Pull request by melroy89**
This update introduces the ability to request the takeover of abandoned magazines.

- List of abandoned magazines (owner hasn't logged in for a month. Melroy: only 1 month??)
- Improve Magazine Voter security for banned users
- Fixes cross-posting
- The ability to submit a request for taking ownership of a warehouse @todo 
- Section in the administrator panel where one can reject or confirm the takeover of a magazine @todo
- Remove `lower` for Mbin.
- Translation updates

PRs:
- https://codeberg.org/Kbin/kbin-core/pulls/1216
- https://codeberg.org/Kbin/kbin-core/pulls/1217
- https://codeberg.org/Kbin/kbin-core/pulls/1218

Co-committed-by: Ernest Wiśniewski <ernestwisniewski2@gmail.com>

### CODE REVIEW

File: config/kbin_routes/magazine.yaml [hunk 1]
```diff
@@ -13,6 +13,16 @@ magazine_purge:
   path: /m/{name}/purge
   methods: [ POST ]
 
+magazine_purge_content:
```
Comment (melroy89) [inline config/kbin_routes/magazine.yaml:16]: These are the two new paths (the rest is auto formatting to the new `.editconfig` standard)

**Comment by melroy89** [comment]
We might need to combine this PR with: https://codeberg.org/Kbin/kbin-core/pulls/1217/

**Comment by melroy89** [comment]
"transfer of ownership" is not a translation string.

**Comment by melroy89** [comment]
@e-five256  As discussed you can fix the 3 files conflicting on the frontend part.

**Comment by melroy89** [comment]
Also including https://codeberg.org/Kbin/kbin-core/pulls/1218 that most likely fixes the `handleCrossposts()` infamous bug.

**Comment by e-five256** [comment]
Posting for posterity; this is the list of followup items I have so far for tackling after this PR:

- hide menu item for abandoned magazines while logged out
- hide request magazine ownership for suspended accounts
- remove ability for mods to ban admin
- un-lower mod requests in magazine panel
- add translations for "crossposted", "by"
  - or consider changing this, maybe discussion to be had on the layout of the component itself
- replace `<td>` with `<th>` in `<thead>` admin users panel
- replace `<td>` with `<th>` in `<thead>` admin ownership requests

not directly related to this PR, but to the syncs before it

- hide suspended accounts from search
- disable inline replies for suspended accounts
- user notification page purge button icon missing

**Review by nobodyatroot — APPROVED** [review]
I've been running this for about a day and don't see any issues, whether that's good or bad... I don't know. If we want to merge this and move on, I'm all for it.

**Comment by melroy89** [comment]
> Posting for posterity; this is the list of followup items I have so far for tackling after this PR:
> 
>     * hide menu item for abandoned magazines while logged out
> 
>     * hide request magazine ownership for suspended accounts
> 
>     * remove ability to request ownership of remote magazines (noted by @nobodyatroot)
> 
>     * remove ability for global mods to ban admin
> 
>     * un-lower mod requests in magazine panel
> 
>     * add translations for "crossposted", "by"
>       
>       * or consider changing this, maybe discussion to be had on the layout of the component itself
> 
>     * replace `<td>` with `<th>` in `<thead>` admin users panel
> 
>     * replace `<td>` with `<th>` in `<thead>` admin ownership requests
> 
> 
> not directly related to this PR, but to the syncs before it
> 
>     * hide suspended accounts from search
> 
>     * disable inline replies for suspended accounts
> 
>     * user notification page purge button icon missing

Some low hanging fruit or other things can be solved already in this PR or? @e-five256  Or what do you want? Feel free to already push the frontened changes hide the abandoned, changing the td to th, etc.

**Comment by e-five256** [comment]
I was a bit worried RTR9 etc will continue to make more changes to these files, causing conflicts on anything I change. Wasn't sure what the plan was for future RTRs etc, if we were going to continue to sync because of recent events. None of the things I listed are blockers to merge as far as I see

**Comment by melroy89** [comment]
Don't worry too much about merge conflicts. I personally find the improvements you were suggesting adding much value to mbin.

**Pull request merged**